aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -1814,8 +1814,8 @@ static Tensor& std_var_out(
     const char* fname, Tensor& result, const Tensor& self,
     at::OptionalIntArrayRef dim, const std::optional<Scalar>& correction_opt,
     bool keepdim, bool take_sqrt) {
-  TORCH_CHECK(self.device().is_cpu() || self.device().is_cuda(),
-              "std and var only supports tensors on a CPU or CUDA device, but got: ",
+  TORCH_CHECK(self.device().is_cpu() || self.device().is_cuda() || self.device().is_xpu(),
+              "std and var only supports tensors on a CPU or CUDA/XPU device, but got: ",
```

Comment:
How about?
```suggestion
              "std and var supports tensors on CPU, CUDA, or XPU devices only, but got: ",
```

Reply:
I have modified the message here, thanks for the suggestions!